ci/ci-run.sh

```diff
@@ -4,21 +4,26 @@ set -e -o pipefail -u -x
 
 DIRECTORY_HASH=$(git rev-parse HEAD:ci/tox)
 export DIRECTORY_HASH
-cd ci/tox/
+image_name=$1
+cd ci/$image_name/
 
 # If there is no cached image for the expected Git&Python versions and the current state of ci/tox,
 # build the image and push it to the Docker Hub.
-docker-compose --no-ansi pull tox
+docker-compose --no-ansi pull $image_name
```

Comment:
While you're here, pls fix the following warn from `docker-compose` logs (see in the CI): `--no-ansi option is deprecated and will be removed in future versions. Use '--ansi never' instead.`

Reply:
Plus also pass this flag to other invocations of `docker-compose` in **CI scripts** (local runs are fine), to avoid weird formatting in the output like:

![image](https://user-images.githubusercontent.com/3383210/130482804-c699e710-eeb2-40dd-8a74-5844938e8c64.png)